pkg/console/operator/operator.go

```diff
@@ -251,3 +260,21 @@ func (c *consoleOperator) removeConsole(cr *operatorsv1.Console) error {
 
 	return utilerrors.FilterOut(utilerrors.NewAggregate(errs), errors.IsNotFound)
 }
+
+func (c *consoleOperator) handleConfigMapSync(operatorConfig *operatorsv1.Console) error {
+	// Get Operator Config here
+	logoName := operatorConfig.Spec.Customization.CustomLogoFile.Name
+	// add syncing for the custom logo config map
+	if logoName != "" {
```

Comment:
If this is empty, you need to set the destination to source to `""/""`

Reply:
I am skipping the sync code completely if there is no logoName set so I don't know what good it would be to set the source to "/".  Currently, the code is working as intended.

Reply:
`""/""` meant "namespace / name"

Reply:
Still not following here, if the logoName is empty (the value has not been set on the operator config), then I would end up with `"namespace /  "`, plus this code would not get executed.

Reply:
Ensure it deletes?  I was able to easily get about a dozen new configmaps in the`openshift-console` namespace.  That shouldn't happen.